docs/reference/inference/put-inference.asciidoc

```diff
@@ -6,10 +6,11 @@ experimental[]
 
 Creates a model to perform an {infer} task.
 
-IMPORTANT: The {infer} APIs enable you to use certain services, such as ELSER, 
-OpenAI, or Hugging Face, in your cluster. This is not the same feature that you 
-can use on an ML node with custom {ml} models. If you want to train and use your 
-own model, use the <<ml-df-trained-models-apis>>.
+IMPORTANT: The {infer} APIs enable you to use certain services, such as built-in 
+{ml} models (ELSER, E5), models uploaded through Eland, Cohere, OpenAI, or 
+Hugging Face, in your cluster. This is not the same feature that you can use on 
+an ML node with custom {ml} models. If you want to train and use your own model, 
+use the <<ml-df-trained-models-apis>>.
```

Comment:
I think we should reword this. For some of the services (ELSER, text_embedding), this is actually the same feature as the trained models API, and the trained models deployed through the inference service will appear in that API/UI.

Maybe something like:
For built-in models and models uploaded though Eland, the {infer} APIs offer an alternative way to use and manage trained models. However, if you do not plan to use the {infer} APIs to use these models or if you want to use non-NLP models, you should directly use the  \<<ml-df-trained-models-apis\>>